bin/anthology/utils.py

```diff
@@ -14,15 +14,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from lxml import etree
-from urllib.parse import urlparse
-from xml.sax.saxutils import escape as xml_escape
 import itertools as it
 import logging
+import os
 import re
 
-from .people import PersonName
-from . import data
+from lxml import etree
+from urllib.parse import urlparse
+from xml.sax.saxutils import escape as xml_escape
+from typing import Tuple, Optional
+
+from anthology.people import PersonName
+from anthology import data
```

Comment:
What's the reason for this change? It doesn't seem right to me?

Reply:
I had the idea that there was a move from dot-style relative imports to using absolute imports, even within a package, based on an import problem I had a while ago that was resolved by making this change. But I can't find any documentation for this so I'll revert it.